tools/ui-components/src/button/button.test.tsx

```diff
@@ -56,22 +56,6 @@ describe('<Button />', () => {
     expect(button).not.toHaveAttribute('disabled', 'true');
   });
 
-  it('should not trigger the onClick prop if the button is disabled', () => {
-    const onClick = jest.fn();
-
-    render(
-      <Button disabled onClick={onClick}>
-        Hello world
-      </Button>
-    );
-
-    const button = screen.getByRole('button', { name: /hello world/i });
-
-    userEvent.click(button);
-
-    expect(onClick).not.toBeCalled();
-  });
-
```

Comment:
removed this, because we don't need the function to not be called when it's disabled, but need to make the function not do anything similar to how we were doing it in the form submission, and going the old way was causing bugs.